py/src/pathfinder_worker/call.py

```diff
@@ -802,6 +804,17 @@ async def simulate_account_tx(
     return tx_info
 
 
+def allowed_libfuncs_file_for_chain_id(chain_id: StarknetChainId) -> str:
+    if chain_id == StarknetChainId.MAINNET:
+        return "mainnet_libfuncs"
+    elif chain_id == StarknetChainId.TESTNET:
+        return "testnet_libfuncs"
+    elif chain_id == StarknetChainId.TESTNET2:
+        return "testnet2_libfuncs"
+    else:
+        raise ValueError("Invalid chain id")
```

Comment:
Doesn't this change how poeple can use this now? Shoudn't it still be experimental?

Reply:
I think it's more accurate now. We're now using the allow list that's packaged with `cairo-lang`, which hopefully matches the allowed libfuncs list on the sequencer.